polytracker/build.py

```diff
@@ -163,33 +163,36 @@ def _instrument_bitcode(
     DFSAN_ABI_LIST_PATH: Path = _ensure_path_exists(
         _compiler_dir_path() / "abi_lists" / "dfsan_abilist.txt"
     )
-
+    # load pass plugin and specify a pass pipeline
     cmd = [
         "opt",
-        "-enable-new-pm=0",
         "-load",
         str(POLY_PASS_PATH),
-        "-ptrack",
-        f"-ignore-list={POLY_ABI_LIST_PATH}",
+        "-load-pass-plugin",
+        str(POLY_PASS_PATH),
+        "-passes=pt-taint,pt-ftrace,dfsan,pt-rm-fn-attr",
     ]
 
-    if no_control_flow_tracking:
-        cmd.append("-no-control-flow-tracking")
+    # if no_control_flow_tracking:
+    #     cmd.append("-no-control-flow-tracking")
```

Comment:
The flag isn't used in the new `pt-taint` nor `pt-track` pass, but it's used in the old `ptrack` one. So I left it in there in case someone wants to run it.

Reply:
Ahh.. so we have an old version of everything as well??? Is that what makes me confused about the `polytracker_start` vs `taint_start` ??

Reply:
Yes. It's not a great state for the codebase to be in, but I find it easier when referencing the old version for functionality when I write the new version. I think once I have calls to/from uninstrumented code solved, I'll remove the old pass altogether.